src/main/resources/db/migration/common/V1_387__add_arm_rpo_pending_ors.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO object_record_status (ors_id,ors_description) VALUES (nextval('ors_seq'),'Arm RPO Pending');
```

Comment:
Should we add id here to ensure the id matches that within the enum?

Reply:
done